src/bin/pgcopydb/ld_apply.c

```diff
@@ -394,16 +385,16 @@ stream_apply_sync_sentinel(StreamApplyContext *context)
 	/*
 	 * If we know we reached endpos, then publish that as the replay_lsn.
 	 */
-	if (context->reachedEndPos)
+	if (context->reachedEndPos || !findDurableLSN)
 	{
 		durableLSN = context->previousLSN;
 	}
 	else
 	{
 		if (!stream_apply_find_durable_lsn(context, &durableLSN))
 		{
-			log_debug("Skipping sentinel replay_lsn: "
-					  "failed to find a durable LSN matching current flushLSN");
+			log_warn("Skipping sentinel replay_lsn udpate: "
```

Comment:
```suggestion
			log_warn("Skipping sentinel replay_lsn update: "
```

Reply:
Applied locally, will be in the next push. Thanks!